synapse/rest/admin/users.py

```diff
@@ -94,7 +95,14 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
 
         user_id = parse_string(request, "user_id")
         name = parse_string(request, "name")
+
         guests = parse_boolean(request, "guests", default=True)
+        if self._msc3861_enabled and guests:
+            raise SynapseError(
+                HTTPStatus.BAD_REQUEST,
+                "The guests parameter is not supported when MSC3861 is enabled.",
+                errcode=Codes.INVALID_PARAM,
+            )
```

Comment:
Synapse is no longer authoritative whether a user is guest or not. We don't have proper guest semantics in MAS for now though, this might get a few iterations still